content/en/docs/tutorials/_index.md

```diff
@@ -57,6 +57,10 @@ Before walking through each tutorial, you may want to bookmark the
 
 * [Using Source IP](/docs/tutorials/services/source-ip/)
 
+## Migrating to Kubernetes from compose
+
+* [Overview](/docs/tutorials/compose-migration/)
```

Comment:
If we make the following change, it leaves room for other tutorials about migrating to Kubernetes:
```suggestion
## Migrating to Kubernetes

* [Migrating from Docker Compose workflows to Kubernetes](/docs/tutorials/compose-migration/)
```
(also, I _think_ there is only one relevant implementation of Compose - but I might be wrong)